x/pairing/keeper/scores/stake_req.go

```diff
@@ -0,0 +1,29 @@
+package scores
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	epochstoragetypes "github.com/lavanet/lava/x/epochstorage/types"
+)
+
+const stakeReqName = "stake-req"
+
+// StakeReq implements the ScoreReq interface for provider staking requirement(s)
+type StakeReq struct {
+	MinStake sdk.Int
+}
+
+// Score calculates the the provider score as the normalized stake
+func (sr StakeReq) Score(stakeEntry epochstoragetypes.StakeEntry) uint64 {
+	return stakeEntry.Stake.Amount.Quo(sr.MinStake).ToDec().BigInt().Uint64()
+}
+
+func (sr StakeReq) GetName() string {
+	return stakeReqName
+}
+
+// Equal() used to compare slots to determine slot groups.
+// Usually Equal() compares the internal values of the ScoreReq,
+// but StakeReq returns always true because the stake is the score itself
```

Comment:
I think this history is irrelevant going forward,  you can replace with:
``` // Equal always returns true (StakeReq score of an entry is the entry's stake)```

Reply:
done. see https://github.com/lavanet/lava/commit/2cabf1d176e04902c5eb6e41490706fe0783b514